README.md

```diff
@@ -1,56 +1,50 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# OOP school library
 
-> Description the project.
+> Imagine that you are the librarian of OOP University, and you need a tool to record what books are in the library and who borrows them. This app will allow you to:
 
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+- And all of this will be built in a beautiful and well-organized way!
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Ruby
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link]()
```

Comment:
- [ ] Please remove this unused section `live demo` because there is no live demo link yet to make your readme file more professional.

Reply:
Well resolved 👍🏽